CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Improve the speed of group by queries for tables with number of replicas
```

Comment:
Thought this was related to the number of shards per node? Not necessarily related to replicas